napari/_qt/dialogs/qt_reader_dialog.py

```diff
@@ -67,9 +67,25 @@ def setup_ui(self, error_message, readers):
 
         # checkbox to remember the choice (doesn't pop up for folders with no extension)
         if self._extension:
-            self.persist_checkbox = QCheckBox(
-                f'Remember this choice for files with a {self._extension} extension'
-            )
+            if (
+                '*' + self._extension
+                in get_settings().plugins.extension2reader.keys()
+            ):
+                pref = get_settings().plugins.extension2reader[
+                    '*' + self._extension
+                ]
+                warn_message = trans._(
+                    'Override existing preference for *{extension}, "{pref}"',
```

Comment:
Just minor tweak here to use similar language as the initial statement.

```suggestion
                    'Override existing preference for files with a {extension} extension: {pref}',
```